shared/modules/consensus_module.go

```diff
@@ -60,3 +65,11 @@ type ConsensusPacemaker interface {
 	GetPrepareQC() (*anypb.Any, error)
 	GetNodeId() uint64
 }
+
+// This interface represents functions exposed by the Consensus module for StateSync specific business logic.
+// These functions are intended to only be called by the StateSync module.
+type ConsensusStateSync interface {
```

Comment:
Per my other comments above, we might be able to remove this interface.

Reply:
I made them in an interface to underline the functions used by submodules, i.e. pacemaker and state sync, but I don't have strong arguments for keeping it, especially for state sync.